diff --git a/node_modules/react-custom-roulette-r19/dist/utils.js b/node_modules/react-custom-roulette-r19/dist/utils.js
index 3704b06..7a1b741 100644
--- a/node_modules/react-custom-roulette-r19/dist/utils.js
+++ b/node_modules/react-custom-roulette-r19/dist/utils.js
@@ -3,12 +3,8 @@ export var getRotationDegrees = function (prizeNumber, numberOfPrizes, randomDif
     if (randomDif === void 0) { randomDif = true; }
     var degreesPerPrize = 360 / numberOfPrizes;
     var initialRotation = 43 + degreesPerPrize / 2;
-    var randomDifference = (-1 + Math.random() * 2) * degreesPerPrize * 0.35;
     var perfectRotation = degreesPerPrize * (numberOfPrizes - prizeNumber) - initialRotation;
-    var imperfectRotation = degreesPerPrize * (numberOfPrizes - prizeNumber) -
-        initialRotation +
-        randomDifference;
-    var prizeRotation = randomDif ? imperfectRotation : perfectRotation;
+    var prizeRotation =  perfectRotation;
     return numberOfPrizes - prizeNumber > numberOfPrizes / 2
         ? -360 + prizeRotation
         : prizeRotation;
